src/components/common/view/AddWebApp.web.js

```diff
@@ -16,10 +16,9 @@ import addAppIlustration from '../../../assets/addApp.svg'
 import Icon from '../view/Icon'
 import userStorage from '../../../lib/gundb/UserStorage'
 import API from '../../../lib/API/api'
-
 import Text from '../../common/view/Text'
-
 import logger from '../../../lib/logger/pino-logger'
+import Config from '../../../config/config'
 
```

Comment:
General notices

1. functions inside AddWebApp could be wrapped to useCallback
2. probably we can just move them out component as store isn't component prop and can be accessed globally
3. we're having an component doesn't renders anything. probably better is to create an custom hook instead ?

Reply:
we wont gain much. the component is hardly re-rendered if at all, its very top level.
but you can add this as a future improvement in icebox